package com.xquare.domain.meal.exception

internal class DatabaseMealNotFoundException : RuntimeException("Meal from database not found.")
